component/card/card.css

```diff
@@ -0,0 +1,123 @@
+.card {
+    width: 30%; 
+    min-height: 40rem;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    position: relative;
+    border: 2px solid var(--PRIMARY-COLOR);
+}
+
+.discount{
+    color: var(--SECONDARY-COLOR);
+}
+
+.card-shadow{
+    box-shadow: 2px 2px 14px 3px var(--PRIMARY-COLOR);
+}
+
+.card-image {
+    width: 100%;
+    height: 20rem;
+    object-fit: cover;
+}
+
+.card-text-highlight {
+    color: var(--SECONDARY-COLOR);
+}
+
+.card > h1, .card > h2, .card > p, .card > button {
+    padding-left: 1rem;
+}
+
+.card > button {
+    margin: 0 2rem;
+    margin-top: 0.5rem;
+}
+
+.card-badge {
+    color: var(--BACKGROUND-COLOR);
```

Comment:
is the color variable and declaration correct?
You have used bg color for color and then no variable for background
also can you try to keep css properties grouped or sorted alphabetically

Reply:
Property and naming do not match, but I wanted the background color to be the text color for this element.